molecule/default/tests/test_default.py

```diff
@@ -15,33 +15,35 @@
 @pytest.mark.parametrize("pkg", ["apt-transport-https"])
```

Comment:
`apt-transport-https` was deprecated in Ubuntu 20.04 and above, and Debian starting with the 1.5.x version of the apt package, which is included in Debian 10 (Buster) and newer:
```console
➤ apt show apt-transport-https
Package: apt-transport-https
Version: 2.2.4
Priority: optional
Section: oldlibs
Source: apt
Maintainer: APT Development Team <deity@lists.debian.org>
Installed-Size: 166 kB
Depends: apt (>= 1.5~alpha4)
Tag: role::shared-lib
Download-Size: 160 kB
APT-Sources: http://deb.debian.org/debian bullseye/main arm64 Packages
Description: transitional package for https support
 This is a dummy transitional package - https support has been moved into
 the apt package in 1.5. It can be safely removed.
 ```

Reply:
I checked and I believe all Debian-based operating systems we currently support have an appropriate version of `apt` available (and the same language about `apt-transport-https`). We _did_ need it for Debian Stretch previously and as the package still existed for later versions I didn't dig deeply into if it was strictly necessary. That being the case I believe we can probably remove installing this package, which probably renders this PR unnecessary.

Reply:
I think the "Support (but ignore) non-Debian platforms" portion of the PR is still desired, so that this role can be used as a dependency in other Ansible roles.

Reply:
Please see commit 021a522.  I will do a grand cleanup of the commit history once the PR is approved.